packages/custom-elements/ts_src/custom-elements.ts

```diff
@@ -43,7 +43,7 @@ function installPolyfill() {
   const customElements = new CustomElementRegistry(internals);
 
   // The main document is associated with the global registry.
-  document.__CE_registry = customElements;
+  (document.__CE_registry as unknown) = customElements;
```

Comment:
I should probably update this so that `document.__CE_registry` is defined as an instance of the polyfilled `CustomElementRegistry`, not the built-in one. The reason this can't be assigned directly is that the polyfilled `CustomElementRegistry`'s `whenDefined`'s returned promise can resolve to `undefined` in some cases when `polyfillDefineLazy` is used. Hopefully in the future we can drop that feature, since I can't find any users on [GitHub](https://github.com/search?q=polyfillLazyDefine&type=code) or [internally](http://cs/polyfillLazyDefine).

Reply:
`__CE_registry` is now defined using the polyfill type.